apps/studio/components/interfaces/Storage/VectorBuckets/CreateVectorTableSheet.tsx

```diff
@@ -165,16 +162,16 @@ export const CreateVectorTableSheet = ({ bucketName }: CreateVectorTableSheetPro
           projectRef: project.ref,
           connectionString: project?.connectionString,
           serverName: wrapperInstance.server_name,
-          sourceSchema: getVectorBucketFDWSchemaName(values.bucketName),
-          targetSchema: getVectorBucketFDWSchemaName(values.bucketName),
-          schemaOptions: [`bucket_name '${values.bucketName}'`],
+          sourceSchema: getVectorBucketFDWSchemaName(bucketName),
+          targetSchema: getVectorBucketFDWSchemaName(bucketName),
+          schemaOptions: [`bucket_name '${bucketName}'`],
```

Comment:
Changes related to removing bucket name as a field. From what I can see, bucket name is still required upon table creation. If it’s missing, it’s likely a loading state issue, not a user error.